tests/test_mention_timeline.py

```
@@ -73,7 +73,21 @@ def test_timeline_with_filters():
     finally:
         os.unlink(path)
 
+
+def test_timeline_uses_fts_table_name_in_match_clause():
+    path = create_test_db()
+    try:
+        Config.DB_PATH = Path(path)
+        result = get_mention_timeline(keyword="John Doe", granularity="month")
+        assert result["total_matches"] > 0, f"Expected timeline matches, got {result}"
+        assert any(period.startswith("2015-") for period in result["timeline"].keys()), (
```

Ruff (0.15.15)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)